docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -132,7 +147,7 @@ Estimate requests are not expected after the following events.
 Estimate calls will also be made in the control panel when using flows like the following.
 
 * Line-item refund flows
-* Test connection functionality when users navigate to **Settings > Setup > Tax > _TaxProviderName_** in an active MSF-enabled BigCommerce control panel
+* Test connection functionality when merchants navigate to **Settings > Setup > Tax > _TaxProviderName_** in an active MSF-enabled BigCommerce control panel
```

Comment:
Minor question, should this be changed to merchants or **both** merchants and partners? (i.e. do partner stores have the same UI as merchant stores?)

Reply:
Got my answer [here](https://github.com/bigcommerce/dev-docs/pull/1728#discussion_r1152568576)